.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "editor.formatOnSave": true,
+    "editor.renderWhitespace": "boundary",
+    "editor.rulers": [
+        80
+    ],
```

Comment:
🤷‍♂️ I find both of these previous rules to be useful but maybe others wouldn't? We could just those, as well.

Reply:
I  think that these are user preferences rather than workspace mandates.

If the ruler setting were used for enforcement of that width, then it should stay as it's a project coding style issue, but if it's just a UI nicety, definitely should be removed and left for user preference.

Also, we should consider having an `extensions.json` to specify recommended extensions for working on this project (a different PR, though).